### PR TITLE
Init page

### PR DESCRIPTION
**Pull request by dvdn**
Init minimal page content with style

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,38 +4,62 @@
...
         </header>
     </div>
 
     <!-- MAIN CONTENT -->
     <div id="main_content_wrap" class="outer">
       <section id="main_content" class="inner">
+          <a href="https://github.com/CanalTP/navitia" class="project" target="_blank">
+        <h2>Navitia</h2>
+        <p class="language">C++, Python</p>
+        <p>Chosen is a JavaScript plugin that makes long, unwieldy select boxes much more user-friendly.</p>
```
Comment (TeXitoi) [inline index.html]: ???

Comment (TeXitoi) [inline index.html]: and bad indentation

File: index.html [hunk 2]
```diff
@@ -4,38 +4,62 @@
...
+          </a>
+
+          <a href="https://github.com/CanalTP/navitia-playground" class="project" target="_blank">
+            <h2>Navitia Playground</h2>
+            <p class="language">PHP</p>
```
Comment (TeXitoi) [inline index.html]: Nop, that's pure javascript/jquery

File: index.html [hunk 3]
```diff
@@ -4,38 +4,62 @@
...
+            <p>Web UI for the navitia API. You can get a token here and then use this UI here.</p>
```
Comment (TeXitoi) [inline index.html]: remove 'You can get a token here and then use this UI here.'

File: index.html [hunk 4]
```diff
@@ -4,38 +4,62 @@
...
+
+          <a href="https://github.com/CanalTP/Chaos" class="project" target="_blank">
+            <h2>Chaos</h2>
+            <p class="language">Python</p>
+            <p>Chaos is the web service which can feed Navitia with real-time disruptions. It can work together with Kirin which can feed Navitia with real-time delays.</p>
+          </a>
 
+          <a href="https://github.com/CanalTP/MttBundle" class="project" target="_blank">
+            <h2>TimeTable</h2>
+            <p class="language">Symfony</p>
```
Comment (TeXitoi) [inline index.html]: PHP/Symfony ?

File: stylesheets/stylesheet.css [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+h4 {
+    font-size: 1.4rem;
+    font-weight: normal; }
+
+h5 {
+    font-size: 1rem;
+    font-weight: normal; }
+
+p {
+  margin: 0 0 20px; }
```
Comment (TeXitoi) [inline stylesheets/stylesheet.css]: margin with 3 args?

Comment (TeXitoi) [inline stylesheets/stylesheet.css]: pixel sizing?

File: stylesheets/stylesheet.css [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+
+section#main_content {
+    width: 90%;
+    margin: auto;
+}
+
+.project {
+    display: block;
+    width: 200px;
+    height: 200px;
```
Comment (TeXitoi) [inline stylesheets/stylesheet.css]: pixel sizing?

**Comment by TeXitoi** [comment]
Laking at least kirin and mimirsbrunn.

File: stylesheets/stylesheet.css (hunk 5 above)
Comment (dvdn) [inline stylesheets/stylesheet.css]: yes, margin with 3 args =  the top is set to the first value, the left and right are set to the second, and the bottom is set to the third.

File: stylesheets/stylesheet.css (hunk 6 above)
Comment (dvdn) [inline stylesheets/stylesheet.css]: For pixel sizing, I honestly don't know what's best choice.
Final rendering looks ok for the moment.

File: index.html (hunk 2 above)
Comment (dvdn) [inline index.html]: My mistake :stuck_out_tongue:

File: stylesheets/stylesheet.css (hunk 5 above)
Comment (TeXitoi) [inline stylesheets/stylesheet.css]: I didn't know that.

File: index.html [hunk 7]
```diff
@@ -1,43 +1,90 @@
...
+                <p>Web UI for the navitia API.</p>
+            </a>
 
-      </section>
-    </div>
+            <a href="https://github.com/CanalTP/Chaos" class="project" target="_blank">
+                <h2>Chaos</h2>
+                <p class="language">Python</p>
+                <p>Chaos is the web service which can feed Navitia with real-time disruptions. It can work together with
+                    Kirin which can feed Navitia with real-time delays.</p>
```
Comment (antoine-de) [inline index.html]: Kirin is referenced here but does not appear anywhere, I think we should add a section about it

**Pull request merged**